lib/index.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
-const zstd = require('bindings')('zstd');
+const { resolve } = require('path');
 const { promisify } = require('util');
 const { isUint8Array } = require('util/types');
 
+const zstd = require(resolve(__dirname, '../build/Release/zstd.node'));
```

Comment:
What about attempting an import of the Debug version?

Reply:
It's not something to be "set up" you just add a flag:

```sh
npx node-gyp rebuild --debug 
```

I don't think we'd ever ship Debug builds if that's what you mean by "set up". What are you imagining for that?